src/store/plugins/topic-change-management.plugin.js

```diff
@@ -24,47 +25,86 @@ let isFirstSetTopic = true
  * @param store
  */
 const topicChangeManagementPlugin = (store) => {
-    store.subscribe((mutation, state) => {
+    store.subscribe((mutation) => {
         // we listen to the "change topic" mutation
-        if (mutation.type === CHANGE_TOPIC_MUTATION) {
-            const currentTopic = state.topics.current
-            // we only set background (from topic) after app startup
-            // otherwise, the URL param bgLayer is ignored/overwritten by the setTopic
-            // we set it anyway if the URL doesn't contain a bgLayer URL param
-            if (!isFirstSetTopic || window.location.href.indexOf('bgLayer=') === -1) {
+        if (
+            // During application startup we trigger a changeTopic before the topics are loaded
+            // therefore in this case we ignore the changeTopic event
+            (mutation.type === 'changeTopic' && store.state.topics.config.length > 0) ||
+            mutation.type === 'setTopics'
+        ) {
+            let startTime = performance.now()
```

Comment:
I know this is quite helpful to debug and improve performance, but shouldn't we check on which ENV we are to start "profiling" this mutation?
We don't really want this kinds of logs to permeate to a production environment

I don't know if it wouldn't be a better solution to implement some kind of "start-stop" logging in the log utils directly, and tie the creation of the `performance.now()` to the env we are currently on.

Reply:
I wanted to remove this I forgot it 😉